js-exercises/array-cube-root-to-json/arrayCubeRootToJson.js

```diff
@@ -0,0 +1,14 @@
+const arrayCubeRootToJson = arr => {
+  if (!Array.isArray(arr)) throw new Error("Not an array");
+
+  if (arr.some(val => isNaN(val))) throw new Error("Wrong data in array");
```

Comment:
expect(() => { arrayCubeRootToJson([false]); }).toThrow();

try this test case. So isNaN gives false for booleans, it treats them as numbers.  Read this article
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN

Reply:
Thanks to inform